keg_elements/db/mixins.py

```diff
@@ -19,6 +20,26 @@
 might_flush = decor.keyword_optional('_flush', after=dbutils.session_flush)
 
 
+@wrapt.decorator
+def kwargs_match_entity(wrapped, instance, args, kwargs):
+    """
+    Asserts that the kwargs passed to the wrapped method match the columns/relationships
+    of the entity.
+    """
+    if kwargs.get('_check_kwargs', True):
+        insp = sa.inspection.inspect(instance)
+
+        # Only allow kwargs that correspond to a column or relationship on the entity
+        allowed_keys = {col.key for col in insp.columns} | set(insp.relationships.keys())
```

Comment:
NIT: At some point it would be nice to have this as a function on the entities. Something like `sa_fields` so you could do `instance.sa_fields` and get a `set` of all the values you can update. 

We end up building this list enough that I could see it being helpful